doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -180,6 +180,10 @@ Bluetooth libraries and services
 
   * Added units for :c:struct:`bt_rscs_measurement` members.
 
+* :ref:`ble_rpc` library:
+
+  * Fixed the issue related to missing buffer size variables for the user phy update and the user data length update procedures.
```

Comment:
Is `phy` referring to physical layer? If so, please spell it out, or spell out what it stands for.

Reply:
Yes it is a short for Physical layer. It is almost always described just PHY. I can use a capital letters here as well. What do you think ?

Reply:
Ah yes, I see that now in other docs. PHY works perfectly!

Reply:
Done :)